packages/server/src/course/course.entity.ts

```diff
@@ -58,4 +58,7 @@ export class CourseModel extends BaseEntity {
   @Exclude()
   // TODO: can we make these not nullable and work with TypeORM
   semesterId: number;
+
+  @Column('boolean', { nullable: true })
+  enabled: boolean;
```

Comment:
what's being enabled here? could do with a better name..

Reply:
This flag sets wether or not a course is using the app. Maybe a rename to `isUsingApp` might be better?

Reply:
I just added a comment to make it more clear